docs/next/guides/integrate-with-angular/angular-language-change-example.md

```diff
@@ -70,3 +74,30 @@ import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 platformBrowserDynamic().bootstrapModule(AppModule).catch(err => { console.error(err) });
 ```
+
+## Related articles
+
+#### Related guides
+
+- [Language](@/guides/internationalization/language.md)
+- [Layout direction](@/guides/internationalization/layout-direction.md)
+- [Locale](@/guides/internationalization/locale.md)
+
+#### Related blog articles
+
+- [Handsontable 12.0.0: RTL support, and a new keyboard shortcuts API](https://handsontable.com/blog/handsontable-12.0.0-data-grid-rtl-support-and-a-new-keyboard-shortcuts-api)
```

Comment:
To be removed

Reply:
Done: [2fd94f9](https://github.com/handsontable/handsontable/pull/9374/commits/2fd94f9415c983ceabfef7165a1debdf1f616360)